core/chains/evm/client/rpc_client.go

```diff
@@ -33,6 +34,47 @@ import (
 	ubig "github.com/smartcontractkit/chainlink/v2/core/chains/evm/utils/big"
 )
 
+//go:generate mockery --quiet --name ChainClientRPC --structname MockChainClientRPC --filename "mock_chain_client_rpc_test.go" --inpackage --case=underscore
+type ChainClientRPC interface {
```

Comment:
I'm noticing some redundancy here between the `ChainClient` interface we declare in chain_client.go and this interface.